parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -239,6 +239,13 @@ try
         Write-Log "Create the Pause Container kubletwin/pause"
         New-InfraContainer -KubeDir $global:KubeDir
 
+        if (-not (Test-ContainerImageExists -Image "kubletwin/pause")) {
+            Write-Log "Could not find container with name kubletwin/pause"
+            $o = docker image list
+            Write-Log $o
+            throw "Kubeletwin/pause container does not exist!"

```

Comment:
kubeletwin/kubletwin - can you make the throw match the actual image name even if its misspelled?

I thought we fixed this to kubeletwin everywhere in some PR but I guess not.

Reply:
Fixed (but had to rebase so comments are stale)